build/yamls/antrea-windows-containerd-with-ovs.yml

```diff
@@ -5,9 +5,29 @@ data:
     mkdir -force c:/var/log/antrea
     $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
     $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
-    mkdir -force C:/var/run/secrets/kubernetes.io/serviceaccount
-    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/ca.crt C:/var/run/secrets/kubernetes.io/serviceaccount
-    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/token C:/var/run/secrets/kubernetes.io/serviceaccount
+
```

Comment:
Why do you manually this (line 8 to line 30) in the generated manifest yaml? I think it is supposed to exist in file `Install-WindowsCNI-Containerd.ps1`, and automatically added into this file when calling `make manifest` ?

Reply:
So any changes to Install-Windows-Containerd.ps1 should only be made in the respective file and not in the yamls and should we be doing the same for Run-Antrea-OVS-Containerd.ps1 as well?
Also , if you suggest the above change can be excluded from this PR as we a;leady have a separate PR for containerd version related changes https://github.com/antrea-io/antrea/pull/5528/files#diff-820e743fbc6549f822a09b2968d46254f0376b3889de673ba58fd4e85da0c8c1

Reply:
I have removed this change for Install-Windows-Containerd.ps1 as we already have another PR for that change